core/src/software/aws/toolkits/core/region/AwsRegion.kt

```diff
@@ -35,3 +35,10 @@ data class AwsRegion(val id: String, val name: String, val partitionId: String)
         private fun String.trimPrefixAndRemoveBrackets(prefix: String) = this.removePrefix(prefix).replace("(", "").replace(")", "").trim()
     }
 }
+
+fun AwsRegion.toEnvironmentVariables(existing: MutableMap<String, String>, replace: Boolean = false) {
```

Comment:
name this `merge[with|Into]EnvironmentVariables`?

Reply:
I wonder if I should flip it around and have :

`MutableMap<String, String>.put(region: AwsRegion, replace: Boolean = false)`

Might be a little less discoverable is all - but that's effectively what we're doing - we're putting the region into the existing variables.

Reply:
/shrug the readability / discoverability may be better but im not sure. Map<String, String> is pretty generic

Reply:
k - will try to come up with a better name